widget/tree.go

```diff
@@ -231,7 +231,41 @@ func (t *Tree) OpenBranch(uid TreeNodeID) {
 	t.Refresh()
 }
 
-// Resize sets a new size for a widget.
+// Opens the branches leading to a node
+//
+// Since: 2.8
```

Comment:
There is no since required for private methods.

Reply:
removed